integr8ly/application-monitoring-operator
# Bump AMO to 1.4.0 and Grafana operator to v3.6.0

## Description
[MGDAPI-410](https://issues.redhat.com/browse/MGDAPI-410)

- Bump Grafana Operator to v.3.6.0
- Bump AMO to v1.4.0

## Verification
- Install locally using this branch
- Ensure the installation completes without error and Grafana is accessable
- In the middleware-monitoring namespace, check to ensure there are no errors in the Grafana pods 
- Validate the correct Grafana Operator (`v3.6.0`) and the AMO versions (`v1.4.0`) are installed
